Conduit/Conduit/Controllers/ArticleController.cs

```diff
@@ -78,12 +77,12 @@ public async Task<ActionResult<List<Article>>> GetAllArticle(int PageNumber)
                 var ArticlesNyumberInPage = 3f;
```

Comment:
it should be const, not var
* typo in Number
* not float should be int, it's number of articles